collectors/ciscoamp/collector.js

```diff
@@ -96,6 +96,11 @@ class CiscoampCollector extends PawsCollector {
         if (state.apiQuotaResetDate && moment().isBefore(state.apiQuotaResetDate)) {
             AlLogger.info(`CAMP000002 API hourly Limit Exceeded. The quota will be reset at ${state.apiQuotaResetDate}`);
             state.poll_interval_sec = 900;
```

Comment:
I wonder if we hit API hourly limit does it actually worth trying to make another call in 15 mins. Looks like we will hit throttling issue again.

Reply:
Api return the [x-ratelimit-remaining](https://github.com/alertlogic/paws-collector/blob/master/collectors/ciscoamp/utils.js#L27) and  when it reset (x-ratelimit-reset) ,accordingly we are setting [apiQuotaResetDate](https://github.com/alertlogic/paws-collector/blob/master/collectors/ciscoamp/collector.js#L110) . There is no fix time after which we have to call.